README.md

```diff
@@ -65,54 +65,65 @@ $ webtorrent --help
    \_/\_/ \___|_.__/ \__\___/|_|  |_|  \___|_| |_|\__|
 
 Usage:
-  webtorrent [command] <torrent-id> <options>
+  webtorrent [command] <torrent-id> [options]
 
-Example:
+Examples:
   webtorrent download "magnet:..." --vlc
+  webtorrent "magnet:..." --vlc --player-args="--video-on-top --repeat"
```

Comment:
This doesn't match any command though

Reply:
`{ command: '$0 [torrent-ids...]', handler: (args) => { processInputs(args.torrentIds, runDownload) } },`

this ? Line ~73

Reply:
I mean, it's not in the readme/help command

Reply:
this implies default command:
```
Usage:
  webtorrent [command] <torrent-id> [options]
```

it is possible to add default command description to commands section of the help output (extra entry).
I can also add `[default]` to the description of `download`

Reply:
yup, that'd work